src/xdocs/checks/blocks/rightcurly.xml

```diff
@@ -172,33 +181,40 @@ public class Test {
         </source>
         <p id="Example2-code">Example:</p>
         <source>
-public class Test {
+public class Example2 {
 
   public void test() {
 
+    boolean foo = false;
     if (foo) {
       bar();
-    } else { bar(); }   // violation, right curly must be alone on line
+    } else { bar(); }
+    // violation above, ''}' at column 21 should be alone on a line.'
 
     if (foo) {
-      bar();
+        bar();
     } else {
-      bar();
-    }                   // OK
+        bar();
+    }
 
     try {
       bar();
-    } catch (Exception e) { // OK because config is set to token METHOD_DEF and LITERAL_ELSE
+    } catch (Exception e) { // OK because config is
+      // set to token METHOD_DEF and LITERAL_ELSE
```

Comment:
Can we keep it single line as it was before ?
Apply it to all cases like this

Reply:
exceeding character limit per line for all posible cases. build is failing other wise